superbench/benchmarks/micro_benchmarks/ort_inference_performance.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""TensorRT inference micro-benchmark."""
+
+import time
+import statistics
+from pathlib import Path
+
+import torch
+import torchvision.models
+import numpy as np
+import onnxruntime as ort
+from onnxruntime.quantization import quantize_dynamic
```

Comment:
We already have had onnxruntime. Why not use ort.quantization directly because we just use once.

Reply:
Done.

Reply:
Got it. Make sense!